.devcontainer/devcontainer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "image": "mcr.microsoft.com/devcontainers/python:0-3.11",
+    "image": "mcr.microsoft.com/devcontainers/python:0-3.10",
```

Comment:
add comment why 3.11 doesn't work

Reply:
https://stackoverflow.com/questions/75048688/picklingerror-could-not-serialize-object-indexerror-tuple-index-out-of-range